site/docs/v1/tech/apis/identity-providers/_identity-provider-tenant-configuration.adoc

```diff
@@ -0,0 +1,14 @@
+[field]#identityProvider.tenantConfiguration# [type]#[Map<UUID, Object>]# {optional_tag} [since]#Available since 1.32.0#::
+The configuration for each Tenant that the identity provider is enabled for.
+
+[field]#identityProvider.tenantConfiguration``[tenantId]``.data# [type]#[String]# {optional_tag} [since]#Available since 1.32.0#::
+An object that can hold any information about the identity provider tenant configuration that should be persisted.
+
+[field]#identityProvider.tenantConfiguration``[tenantId]``.limitUserLinkCount# [type]#[Object]# {optional_tag} [since]#Available since 1.32.0#::
+The policy that governs identity provider links on a Tenant.
+
+[field]#identityProvider.tenantConfiguration``[tenantId]``.limitUserLinkCount.enabled# [type]#[Integer]# {optional_tag} [default]#defaults to `false`# [since]#Available since 1.32.0#::
+When enabled, the number of identity provider links a user may create is enforced by [field]#maximumLinks#.
+
+[field]#identityProvider.tenantConfiguration``[tenantId]``.limitUserLinkCount.maximumLinks# [type]#[Integer]# {optional_tag} [default]#defaults to `42`# [since]#Available since 1.32.0#::
+The maximum number of identity provider accounts that a user can link to.
```

Comment:
This configuration is specific to this IdP. So a user can have more links than this number, but for this particular IdP, the number of links will be limited and enforced.